sdk/openai/Azure.AI.OpenAI/tests/Utils/Config/IConfiguration.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework.Internal;
+
+namespace Azure.AI.OpenAI.Tests.Utils.Config;
+
+/// <summary>
+/// A test configuration for an Azure resource.
+/// </summary>
+public interface IConfiguration
```

Comment:
Why not just use the `Microsoft.Extensions.Configuration` abstractions here rather than rolling a custom one?

Reply:
Hmm... For some reason I had always thought of that as an ASP.Net thing even though it turns out it isn't. Since this is just for a single test project (and not a shared framework), and is intended to be a stopgap until System.ClientModel based test framework is ready, I don't think it makes sense to rework this based on that.